_inc/client/components/settings-card/index.jsx

```diff
@@ -75,6 +76,24 @@ export const SettingsCard = props => {
 					/>
 				);
 
+			case FEATURE_WORDADS_JETPACK:
+				if (
+					'is-premium-plan' === planClass ||
+					'is-business-plan' === planClass
```

Comment:
Since `includes()` is already loaded in this file, maybe you could do
`includes( [ 'is-premium-plan', 'is-business-plan' ], planClass )`